tests/integration/suite/daprd/actors/actors.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package actors
+
+import (
+	"context"
+	"testing"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/framework/process/exec"
+	"github.com/dapr/dapr/tests/integration/framework/process/logline"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(actors))
```

Comment:
This should live under `/suite/actors`. That package itself is probably in the wrong place and should live under `/suite/daprd/actors` but we shouldn't change that in this PR.

Rename this test (struct) to `emptyaddress` or similar.

Reply:
yeah I agree that the `/suite/actors` should move to `/suite/daprd/actors`. For now, I've changed the struct to be `emptyaddress` per your suggestion.